apm-agent-plugins/apm-rabbitmq/apm-rabbitmq-plugin/src/test/java/co/elastic/apm/agent/rabbitmq/RabbitMQTest.java

```diff
@@ -673,7 +674,9 @@ private static void checkSpanCommon(Span span, String expectedAction, String exp
         Destination destination = span.getContext().getDestination();
 
         assertThat(destination.getPort()).isEqualTo(container.getAmqpPort());
-        assertThat(destination.getAddress().toString()).isEqualTo(container.getHost());
+        if (!System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("win")) {
```

Comment:
What do we get instead on Windows? The host name is just not set?

Reply:
On my machine tests fails with:
```
org.opentest4j.AssertionFailedError: 
expected: "localhost"
but was : "kubernetes.docker.internal"
Expected :"localhost"
Actual   :"kubernetes.docker.internal"
<Click to see difference>
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at co.elastic.apm.agent.rabbitmq.RabbitMQTest.checkSpanCommon(RabbitMQTest.java:687)
	at co.elastic.apm.agent.rabbitmq.RabbitMQTest.checkSendSpan(RabbitMQTest.java:655)
```

Reply:
OK, maybe some quirkiness related to how this container is set up on Docker for Windows, but why kubernetes??
Anyway, I pushed a suggested test, please try it out. If it doesn't work, let's restore your suggestion and use it.